plugins/outline.py

```diff
@@ -0,0 +1,401 @@
+    """
+    Create a clean overlay of the image by removing inner edges
+    :param img: Image
+    :param target_color: Outline color
+    :param outline_width: Outline width
+    """
+    BLACK = (0, 0, 0, 255)
+    WHITE = (255, 255, 255, 255)
+    TRANSPARENT = (0, 0, 0, 0)
+
+    # Convert to grayscale
+    img = img.convert('L')
+
+    # Floodfill approach adopted from
+    # https://learnopencv.com/filling-holes-in-an-image-using-opencv-python-c/
+
+    # Threshold
+    threshold_val = 255 / 2
+    threshold = img.point(lambda p: 255 if p > threshold_val else 0)
+    threshold = threshold.convert("RGB")
+
+    # Inverted threshold
+    inverted = threshold.copy()
+    seed = (0, 0)
+    ImageDraw.floodfill(inverted, seed, WHITE[0:3], thresh=200)
+    inverted = ImageOps.invert(inverted)
+
+    # Combine
+    bitwise_or_add = ImageChops.add(inverted, threshold)
+
+    # Find edges
+    edges = bitwise_or_add.\
+        filter(ImageFilter.FIND_EDGES).\
+        filter(ImageFilter.MaxFilter(outline_width)).\
+        convert("RGBA")
+
+    def _replace_color(img: Image, target_color: tuple,
+                       replace_color: tuple) -> None:
+        """
+        Replace color in image
+        :param img: Image
+        :param target_color: Color to replace
+        :param replace_color: Color to replace with
+        """
+        data = img.getdata()
+        img.putdata(
+            [item if item != target_color else replace_color for item in data])
+
+    _replace_color(edges, BLACK, TRANSPARENT)
+    _replace_color(edges, WHITE, target_color)
+
+    return edges
+
+
+def _outline(outline_sele: str, outline_color: tuple, outline_width: int,
+             scale: int, reps: tuple) -> None:
+    """
+    Outline a selection's representations with a specific color.
+    :param outline_sele: Selection to outline
+    :param outline_color: Color to outline with
+    :param outline_width: Width of outline
+    :param scale: Scale factor for antialiasing
+    :param reps: Representations to outline
+    """
+    try:
+        tmp_scene = "tmp_scene"
+
+        cmd.scene(tmp_scene, "store", quiet=1)
+
+        # Render what we have
+        base_bytes = cmd.png(filename=None, ray=1)
+
+        # Render only whats outlined
+        cmd.hide('everything')
+        for rep in reps:
+            cmd.show(rep, outline_sele)
+
+        # Ray trace edges (we'll remove inner edges later)
+        ray_trace_mode = cmd.get('ray_trace_mode')
+        cmd.set('ray_trace_mode', 2)
+
+        ray_trace_color = cmd.get('ray_trace_color')
+        cmd.set('ray_trace_color', 'white')
+
+        bg_color = cmd.get('bg_rgb')
+        cmd.bg_color('black')
+
+        ray_opaque_background = cmd.get('ray_opaque_background')
+        cmd.set('ray_opaque_background', 0)
+
+        (width, height) = cmd.get_viewport()
+        overlay_bytes = cmd.png(filename=None, ray=1, width=width*scale,
+                                height=height*scale)
```

Comment:
suggest to turn off PyMOL's built-in [antialias](https://pymolwiki.org/index.php/Antialias) oversampling for this step to mitigate the performance slowdown.